recipe/meta.yaml

```diff
@@ -11,19 +11,24 @@ source:
 
 build:
   number: 0
-  noarch: python
+  skip: True # [py<36]
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
+  build:
```

Comment:
The `build` is not needed for python packages if we do not use compilers or other build tools. `setuptools` should be in `host`